<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicWidget</class>
 <widget class="QDockWidget" name="MosaicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mosaic Tool</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QComboBox" name="comboCoverage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Coverage</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLineEdit" name="textXMax"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="textYMin"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Start date</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboMosaicType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Mosaic type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="sliderStartDate">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>End date</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y max</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QSlider" name="sliderEndDate">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="textXMin"/>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X Max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QLineEdit" name="textYMax"/>
        </item>
        <item row="3" column="5" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonLayerExtent">
            <property name="text">
             <string>Layer extent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonCanvasExtent">
            <property name="text">
             <string>Canvas extent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSelectExtentOnCanvas">
            <property name="text">
             <string>Select on canvas</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3" rowspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCreateMosaic">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Create mosaic</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>135</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
